src/org/usfirst/frc/team2706/robot/controls/TriggerButtonJoystick.java

```diff
@@ -19,17 +20,29 @@
      * @param axis The axis to act as a button
      */
     public TriggerButtonJoystick(Joystick joystick, int axis) {
+        this(joystick, axis, 0.1);
```

Comment:
I'm assuming the 0.1 is the deadzone?    Should be a constant....   :)   This will be useful next year as well!   Great job!